packages/helm-charts/oracle-rbac/values.yaml

```diff
@@ -1,2 +1,4 @@
 environment:
   name: default
+
+oracleCount: 2
```

Comment:
We should probably have the default be 1, and then make sure we override this in celotool using the `--set` syntax. Also for continuity maybe we should call this `oracle.replicas` and do
```
oracle:
  replicas: 1
```